alcs-frontend/src/app/services/application/application.dto.ts

```diff
@@ -66,11 +77,13 @@ export interface SubmittedApplicationParcelDto {
   parcelType: string;
   documentUuids: string[];
   owners: SubmittedApplicationOwnerDto[];
+  documents: ApplicationParcelDocumentDto[];
 }
 
 export interface SubmittedApplicationDto {
```

Comment:
Could probably rename to ApplicationSubmissionDto now

Reply:
done